server/pulp/server/controllers/repository.py

```diff
@@ -699,6 +699,7 @@ def sync(repo_id, sync_config_override=None, scheduled_call_id=None):
     call_config = PluginCallConfiguration(imp_config, repo_importer.config, sync_config_override)
     transfer_repo.working_dir = common_utils.get_working_directory()
     conduit = RepoSyncConduit(repo_id, repo_importer.importer_type_id)
+    conduit.importer_object_id = repo_importer.id
```

Comment:
This is unpleasant and hacky, but I'm not sure if we want to change the RepoSyncConduit __init__ signature. If that's okay, I'd prefer to pass it in and document it there.